tests/unit/aiplatform/test_utils.py

```diff
@@ -360,6 +363,15 @@ def test_client_w_override_select_version():
     )
 
 
+def test_get_timestamp_proto():
```

Comment:
Can we use (or based on) the `get_timestamp_proto` in util?

Reply:
`assert true_timestamp_proto == utils.get_timestamp_proto(time)`

This unit test is testing the `utils.get_timestamp_proto`? I am not quiet sure I understand the question.